src/battle_gfx_sfx_util.c

```diff
@@ -1143,15 +1226,40 @@ void SpriteCB_EnemyShadow(struct Sprite *shadowSprite)
         shadowSprite->callback = SpriteCB_SetInvisible;
         return;
     }
+
+    s8 xOffset = 0, yOffset = 0;
```

Comment:
`yOffset` is unused when using `GEN_3` style.
It'll make the the compile fail when using warnings == errors when compiling